models/comments.models.js

```diff
@@ -0,0 +1,18 @@
+const db = require("../db/connection");
+const checkCommentExists = require("../check-comment-exists");
+
+exports.removeComment = (comment_id) => {
+    return checkCommentExists(comment_id).then((exists) => {
+        if (!exists) {
+            return Promise.reject({
+                status: 404,
+                message: "comment does not exist",
+              });
+        }
+        return db
+        .query("DELETE FROM comments WHERE comment_id = $1;", [comment_id])
+        .then((result) => {
+            return result;
+        })
+    })
```

Comment:
You could avoid the checkCommentExists by simply RETURNING * from the delete then checking the length that comes back. 

However, there is nothing wrong with the logic here. Well done ⭐ 👍